packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -567,13 +567,11 @@ export class ApexDebug extends LoggingDebugSession {
     }
     try {
       if (args.connectType === CONNECT_TYPE_ISV_DEBUGGER) {
-        const config = await new ConfigGet().getConfig(
-          args.salesforceProject,
-          SF_CONFIG_ISV_DEBUGGER_SID,
-          SF_CONFIG_ISV_DEBUGGER_URL
-        );
-        const isvDebuggerSid = config.get(SF_CONFIG_ISV_DEBUGGER_SID);
-        const isvDebuggerUrl = config.get(SF_CONFIG_ISV_DEBUGGER_URL);
+        const configAggregator: ConfigAggregator = await ConfigAggregator.create({
+          projectPath: args.salesforceProject
+        });
+        const isvDebuggerSid = JSON.stringify(configAggregator.getPropertyValue(SF_CONFIG_ISV_DEBUGGER_SID));
```

Comment:
getPropertyValue takes a generic type Param.  so If you know the 2 props you want are string, you can say `getPropertyValue<string>(SF_)`.  the stringify isn't necessary.